x-pack/plugins/elastic_assistant/server/ai_assistant_data_clients/conversations/field_maps_configuration.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FieldMap } from '@kbn/data-stream-adapter';
+
+export const conversationsFieldMap: FieldMap = {
+  '@timestamp': {
+    type: 'date',
+    array: false,
+    required: false,
+  },
+  users: {
+    type: 'nested',
+    array: true,
+    required: false,
+  },
+  'users.id': {
+    type: 'keyword',
+    array: false,
+    required: true,
+  },
+  'users.name': {
+    type: 'keyword',
+    array: false,
+    required: false,
+  },
+  id: {
+    type: 'keyword',
+    array: false,
+    required: true,
+  },
+  title: {
+    type: 'keyword',
+    array: false,
+    required: true,
+  },
+  is_default: {
+    type: 'boolean',
+    array: false,
+    required: false,
+  },
+  updated_at: {
+    type: 'date',
+    array: false,
+    required: false,
+  },
+  created_at: {
+    type: 'date',
+    array: false,
+    required: false,
+  },
+  messages: {
+    type: 'nested',
+    array: true,
+    required: false,
+  },
+  'messages.@timestamp': {
+    type: 'date',
+    array: false,
+    required: true,
+  },
+  'messages.role': {
+    type: 'keyword',
+    array: false,
+    required: true,
+  },
+  'messages.is_error': {
+    type: 'boolean',
+    array: false,
+    required: false,
+  },
+  'messages.content': {
+    type: 'text',
+    array: false,
+    required: false,
+  },
+  'messages.reader': {
+    type: 'object',
+    array: false,
+    required: false,
+  },
+  'messages.trace_data': {
+    type: 'object',
+    array: false,
+    required: false,
+  },
+  'messages.trace_data.transaction_id': {
+    type: 'keyword',
+    array: false,
+    required: false,
+  },
+  'messages.trace_data.trace_id': {
+    type: 'keyword',
+    array: false,
+    required: false,
+  },
```

Comment:
I'm seeing trace_data being returned from `/internal/elastic_assistant/actions/connector/azure-open-ai/_execute`, but not being persisted to the conversation:

<p align="center">
  <img width="800" src="https://github.com/elastic/kibana/assets/2946766/23aad6cc-88e1-4d60-80d6-0f69eea089f0" />
</p>

Reply:
resolved in commit https://github.com/elastic/kibana/pull/173487/commits/2a923e1da1c6ef513ea64e19051791ca5bb8deda